### PR TITLE
fix(toolkits/pro) Upgrading the Ndoe Version, The version release error is rectified;the tinyvue template adapts to the latest version @opentiny/vue

### PR DESCRIPTION
**Pull request by liberty-zlk**
# PR

#113 

## PR Checklist

Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-cli/blob/main/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

File: packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue [hunk 1]
```diff
@@ -315,9 +317,50 @@
       background: var(--ti-tree-menu-node-hover-bg-color) !important;
     }
   }
+
+  :deep(.tiny-tree-node__wrapper > .is-expanded > .tiny-tree-node__children
```
Comment (fengyon) [inline packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue:321]: 为什么为我们需要特殊配置颜色呢

File: packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue [hunk 2]
```diff
@@ -315,9 +317,50 @@
...
+    background-color: #fff !important;
+  }
+
+  :deep(.tiny-tree-node__content:hover) {
+    background-color: var(--ti-tree-node-content-hover-bg-color) !important;
+  }
+
+  :deep(.tiny-tree-menu__wrap > .tiny-tree-node__wrapper > .is-root > .tiny-tree-node__content 
+  > .tiny-tree-node__content-left .tiny-tree-node__content-box .tree-node-name) {
+    padding: 0 20px !important;
```
Comment (fengyon) [inline packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue]: 这些padding 可不可以去掉，以后opentiny/vue升级了，还得再改

File: packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue (hunk 1 above)
Comment (liberty-zlk) [reply packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue:321]: > 为什么为我们需要特殊配置颜色呢
这里我们使用了@opentiny/vue的Treemenu树形组件，组件升级之后，影响了我们之前的菜单样式，这里对菜单颜色样式进行了二次开发，增加了权重。

File: packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue (hunk 2 above)
Comment (liberty-zlk) [reply packages/toolkits/pro/template/tinyvue/src/components/menu/index.vue]: > 这些padding 可不可以去掉，以后opentiny/vue升级了，还得再改

我们这里的菜单配置了图标，为了菜单展示的更好看，就配置padding值

**Review by fengyon — APPROVED** [review]

**Pull request merged**